src/pkg/control/options.go

```diff
@@ -94,6 +102,29 @@ func DefaultRestoreConfig(timeFormat dttm.TimeFormat) RestoreConfig {
 	}
 }
 
+// EnsureRestoreConfigDefaults sets all non-supported values in the config
+// struct to the default value.
+func EnsureRestoreConfigDefaults(
+	ctx context.Context,
+	rc RestoreConfig,
+) RestoreConfig {
+	if !slices.Contains([]CollisionPolicy{Skip, Copy, Replace}, rc.OnCollision) {
+		logger.Ctx(ctx).
+			With(
+				"bad_collision_policy", rc.OnCollision,
+				"default_collision_policy", Skip).
+			Info("setting collision policy to default")
+
+		rc.OnCollision = Skip
+	}
+
+	if strings.TrimSpace(rc.Location) == RootLocation {
```

Comment:
is this valid? Is it possible for users to have a folder (very adversarial) that's "/ "? What would it look like if they gave that on the CLI and it got to this point?